lisa/schema.py

```diff
@@ -1315,6 +1315,7 @@ class RemoteNode(Node):
     username: str = constants.DEFAULT_USER_NAME
     password: str = ""
     private_key_file: str = ""
+    use_ipv6: bool = False
```

Comment:
I'm thinking if it's confusing, when set it to true for other platforms like ready? Does ipv6 work directly for ready, if set the IP address to ipv6?

Reply:
I tried to set ipv6 for ready platform, it worked with changes in shell.py